docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue, email, or any other method with the owners of this repository
+before making a change.
+
+
+## Development environment setup
+
+To set up a development environment, please follow these steps:
+
+   ```sh
+   git clone https://github.com/starknet-id/starknet.quest.git
+   cd starknet.quest
+   npm i && npm run dev
+   ```
+
+## Issues and feature requests
+
+Issues with the label **open for contribution** are for you to contribute to. These are exclusively for people who are interested in contributing meaningfully to the repository. We welcome your effort in issues which come under this label.
+
+**All issues can be completed within 3 business days so this can be taken as a hard deadline for resolving a particular issue once you start working on it**
```

Comment:
I think we can just mention following the time deadline in the issue otherwise OD reward will be significantly lower 
 and the issue can be given to anyone else.
 
 Wdyt ?

Reply:
done